### ISSUE TITLE
Saving table entity with double property without decimal point is stored as String type

### ISSUE THREAD

**Issue by nirhadassi** [Azure/azure-storage-node#149]
After heavy debugging we found a bug in your npm.
This is the scenario:

A property of type Edm.Double with a value without decimal point is stored, let's say the value is 100.
At `edmhandler.js` at `serializeValue` we switch to case:
```
    case EdmType.DOUBLE:
      if(azureutil.objectIsInt(value)) {
        return value + '.0';
      }      
      return value;
```
so now our value is '100.0'.

Next at `odatahandler.js` we check ` if (edmHandler.isTypeRequired(type, value))`
At `isTypeRequired `we switch to case:
```
  case EdmType.DOUBLE:
    if (value === Number.POSITIVE_INFINITY || value === Number.NEGATIVE_INFINITY || (Number.isNaN(value))) {
      return true;
    }
    return false;
```
So now we get that the type is not required and the value is '100.0' which is stored as String.
When we try to deserialize it as Double at our C# code it breaks.

The expected outcome is storing it as Double, which can be resolved by always returning true for `isTypeRequired `for the Double case.

Please advise,
Nir.

**Comment by hasonmsft** [comment]
@nirhadassi , thanks for raising this issue. This issue has been already found and will be addressed in the next release.

**Comment by nirhadassi** [comment]
Thanks! When can we expect the next release?

**Comment by hasonmsft** [comment]
@nirhadassi , should be before end of this month. Will get you updated when it's released.

**Comment by hasonmsft** [comment]
@nirhadassi , the fix is already in the latest release (1.0.1).

**Comment by nirhadassi** [comment]
@hasonmsft Great! Thanks :)
